### PR TITLE
add ToneAlarm test/example for ChibiOS builds

### PR DESCRIPTION
**Pull request by kd0aij**
Plays each defined tune at 4 second intervals.

also fix a type error in ToneAlarm::play() which did not cause any apparent problems 
(thanks @peterbarker)

tested with fmuv3 build on mRo X2.1

### CODE REVIEW

File: libraries/AP_Notify/examples/AP_ToneAlarm_Chibios_test/AP_ToneAlarm_ChibiOS_test.cpp [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/*
+ *       Example of AC_Notify library .
+ *       DIYDrones.com
```
Comment (amilcarlucas) [inline libraries/AP_Notify/examples/AP_ToneAlarm_Chibios_test/AP_ToneAlarm_ChibiOS_test.cpp]: I think this line should be ardupilot.org

File: libraries/AP_HAL_Linux/ToneAlarm.cpp [hunk 2]
```diff
@@ -81,7 +81,7 @@ void ToneAlarm::stop()
 
 bool ToneAlarm::play()
 {
-    uint16_t cur_time = AP_HAL::millis();
+    uint32_t cur_time = AP_HAL::millis();
```
Comment (amilcarlucas) [inline libraries/AP_HAL_Linux/ToneAlarm.cpp]: it should be const

File: libraries/AP_HAL_ChibiOS/ToneAlarm.cpp [hunk 3]
```diff
@@ -72,7 +72,7 @@ void ToneAlarm::stop()
 
 bool ToneAlarm::play()
 {
-    uint16_t cur_time = AP_HAL::millis();
+    uint32_t cur_time = AP_HAL::millis();
```
Comment (amilcarlucas) [inline libraries/AP_HAL_ChibiOS/ToneAlarm.cpp]: it should be const

**Comment by kd0aij** [comment]
@amilcarlucas Thanks for the review. Added const and removed example so maybe it will pass CI now.

**Comment by amilcarlucas** [comment]
Why did you remove the example? Const is not there.

**Comment by amilcarlucas** [comment]
Why did you remove the example? Const is not there.

**Comment by kd0aij** [comment]
replaced by https://github.com/ArduPilot/ardupilot/pull/8110

**Pull request closed without merging**